spyder/config/main.py

```diff
@@ -206,6 +206,7 @@
               'blank_spaces': False,
               'edge_line': True,
               'edge_line_column': 79,

```

Comment:
Maybe we need to add a comment here to say we are no longer using this, or remove the config and bump the version number of the config as it is described on the bottom of this file.

@ccordoba12 ?